job_definitions/rerelease_all.yml

```diff
@@ -0,0 +1,73 @@
+{% set apps = ['router', 'search-api', 'api', 'antivirus-api', 'admin-frontend', 'briefs-frontend', 'brief-responses-frontend', 'buyer-frontend', 'supplier-frontend', 'user-frontend'] %}
+---
+- job:
+    name: rerelease-all-apps
+    display-name: "Re-release all apps"
```

Comment:
Can we call this `Re-release all apps with latest credentials` so it's super clear what's happening?